specification/maps/data-plane/Geolocation/preview/1.0/geolocation.json

```diff
@@ -56,7 +56,8 @@
     "/geolocation/ip/{format}": {
       "get": {
         "description": "\n**Applies to**: S0 and S1 pricing tiers.\n\n\nThis service will return the ISO country code for the provided IP address. Developers can use this information  to block or alter certain content based on geographical locations where the application is being viewed from.",
-        "operationId": "GetLocation",
+        "operationId": "Geolocation_GetIPToLocation",
+        "x-ms-client-name": "GetLocation",
```

Comment:
To verify, changing the operation ID like this would relocate these operations from the Client to a GeolocationOperations object. Does this x-ms-client-name preserve that? 

I would expect this would generate `GeolocationOperations.get_location`. Is that correct?

Reply:
@tjprescott Yes, this `"x-ms-client-name": "GetLocation"` will still preserve the GeolocationOperations object so the method signature matches what you expect. 

I've also tried passing `--inspector` options to the autorest, and found that it is the code model generated by the modelerfour that preserves the operationGroup.

Reply:
thanks @alankashiwa 

@tjprescott if there aren't any more concerns, can we please get this one merged? We'd like to start working on convenience layer next week. thanks!